sysbuild/netcore.cmake

```
@@ -41,22 +41,45 @@ if(SB_CONFIG_NETCORE_802154_RPMSG)
   if(NOT "${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN}" IN_LIST PM_DOMAINS)
     list(APPEND PM_DOMAINS ${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN})
   endif()
-  list(APPEND PM_${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN}_IMAGES
-      "${SB_CONFIG_NETCORE_802154_RPMSG_NAME}"
-  )
   set_property(GLOBAL APPEND PROPERTY
-      PM_${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN}_IMAGES
-      ${PM_${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN}_IMAGES}
+               PM_${SB_CONFIG_NETCORE_802154_RPMSG_DOMAIN}_IMAGES
+               "${SB_CONFIG_NETCORE_802154_RPMSG_NAME}"}
```

Caused a syntax warning:
```suggestion
               "${SB_CONFIG_NETCORE_802154_RPMSG_NAME}"
```

---

Fixed